pkgs/applications/networking/cluster/k3s/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, callPackage }:
+
+let
+  # The 1_26 builder can compile 1.26, and hopefully newer versions
+  k3s_1_26_builder = import ./builders/1_26.nix;
+in
+{
+  k3s_1_26 = (callPackage k3s_1_26_builder { }) {
+    k3sVersion = "1.26.4+k3s1";
+    k3sCommit = "8d0255af07e95b841952563253d27b0d10bd72f0";
+    k3sRepoSha256 = "0qlszdnlsvj3hzx2p0wl3zhaw908w8a62z6vlf2g69a3c75f55cs";
+    k3sVendorSha256 = "sha256-JXTsZYtTspu/pWMRSS2BcegktawBJ6BK7YEKbz1J/ao=";
+    chartVersions = import ./1_26/chart-versions.nix;
+    k3sRootVersion = "0.12.1";
+    k3sRootSha256 = "0724yx3zk89m2239fmdgwzf9w672pik71xqrvgb7pdmknmmdn9f4";
+    k3sCNIVersion = "1.1.1-k3s1";
+    k3sCNISha256 = "14mb3zsqibj1sn338gjmsyksbm0mxv9p016dij7zidccx2rzn6nl";
+    containerdVersion = "1.6.19-k3s1";
+    containerdSha256 = "12dwqh77wplg30kdi73d90qni23agw2cwxjd2p5lchq86mpmmwwr";
+    criCtlVersion = "1.26.0-rc.0-k3s1";
+  };
+
+  # 1_27 can be built with the same builder as 1_26
+  k3s_1_27 = (callPackage k3s_1_26_builder { }) (import ./1_27/versions.nix) // {
```

Comment:
I think this name is a bit weird. Using `k3s_1_26_builder` to build `k3s_1_27` feels awkward.

I'd call it just `builder.nix` because chances are that any new versions above of 1.26 can keep being built with the same builder and different version inputs. Even the update script can probably be shared.

But these are just details. Overall it looks a very good change to me.